articles/libraries/machine-learning/index.md

```diff
@@ -10,4 +10,12 @@ uid: microsoft.quantum.machine-learning.concepts.intro
 
 # Introduction to the Quantum Machine Learning Library
 
-The purpose of this documentation is to provide a concise introduction to machine learning tools (written in Q\#) for hybrid quantum/classical learning. We begin with a brief introduction to machine learning concepts and specifically their realization in quantum circuit centric classifiers. We then discuss the training and validation methods for such classifiers and how they map on specific Q\# operations provided by the library.
\ No newline at end of file
+The Quantum Machine Learning Library is a high-level API, written in Q#, that provides the first playground to run hybrid quantum/classical machine learning experiments. From this library you can expect:
+
+- Basic tools to load your own data to classify with quantum simulators
```

Comment:
I would list the features of the library here, and put the limitations in a paragraph below